samples/bluetooth/fast_pair/locator_tag/README.rst

```diff
@@ -135,11 +135,17 @@ To start Fast Pair discoverable advertising after the FMDN unprovisioning and fa
 Device Firmware Update (DFU)
 ============================
 
-The locator tag sample uses the :ref:`MCUboot <mcuboot:mcuboot_ncs>` bootloader firmware image, enabling application firmware image upgrades through the :ref:`app_dfu` procedure.
-Over-the-air updates are supported using MCUmgr's Simple Management Protocol (SMP) over Bluetooth.
+The locator tag sample supports over-the-air updates using MCUmgr's Simple Management Protocol (SMP) over Bluetooth.
+The application configures the appropriate DFU solution in relation to the selected build target (see the table below for more details).
+The following DFU solutions are supported in this sample:
+
+* The :ref:`MCUboot <mcuboot:mcuboot_ncs>` bootloader solution.
+  See the :ref:`app_dfu` user guide for more information.
+* The Software Update for Internet of Things (SUIT) solution.
+  See the :ref:`SUIT <ug_nrf54h20_suit_dfu>` user guide for more information.
 
 To enable the DFU functionality use the ``SB_CONFIG_APP_DFU`` sysbuild Kconfig option.
-It is enabled by default if the MCUboot bootloader image is used.
+This option is enabled by default if the build target configuration has enough memory to support it (see the table below to learn about unsupported configurations).
```

Comment:
```suggestion
This option is enabled by default if the build target configuration has enough memory to support it (see the following table to learn about supported configurations).
```
Since the table describes about different configurations.